packages/workflow/src/records/fhir.ts

```diff
@@ -245,7 +245,8 @@ export function createCorrectedTask(
           }
         ]
       },
-      valueString: update.newValue
+      newValueString: update.newValueString,
```

Comment:
We need to use keys here that conform with the FHIR spec

https://fhir-ru.github.io/extensibility.html#:~:text=The%20value%5Bx%5D%20element%20has,valueMeta%3A%20Meta